### PR TITLE
Updating README.md, and displaying both °C and °F.

### PR DESCRIPTION
**Pull request by Tim-Abwao**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,50 @@
...
+* working with JSON.
 * modular programming
 
-When a client visits the application, it
+When a client visits the application, it:
+
 1. gets the client's IP address.
-1. uses the IP address to look up their location.
-1. uses their location data to greet them with temperature of the city they are located in.
+2. uses the IP address to look up their location.
```
Comment (xofbd) [inline README.md:15]: There's no need to make this change. In markdown, numbered lists are created using `1.` but when rendered, the appropriate numbers are used. That way, there's no need to update the numbering when changes are made. A little confusing, I know.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,50 @@
...
+3. uses their location data to greet them with temperature of the city they are located in.
 
 ## Prerequisites
-All required Python packages can be found in the `requirements.txt` file. Additionally, the provided `Makefile` can be used to created a virtual environment by running `make venv`. You will also need a Heroku account and have installed the Heroku CLI. For more information on the Heroku CLI, go to https://devcenter.heroku.com/articles/heroku-cli#download-and-install.
+
+All required Python packages can be found in the `requirements.txt` file. Additionally, the provided `Makefile` can be used to create a virtual environment by running `make venv`. You will also need a Heroku account, and to have installed the [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli#download-and-install).
```
Comment (xofbd) [inline README.md:20]: I like this suggestion.

File: README.md [hunk 3]
```diff
@@ -1,38 +1,50 @@
...
-1. `set FLASK_APP=weather_app/app.py` in the command line.
-1. run `python -m flask run`.
-1. open the link in the command line.
+
+        py -3 -m venv venv
+        venv\Scripts\activate.bat
+
+2. `set FLASK_APP=weather_app/app.py` in the command line.
+3. run `python -m flask run`.
+4. open the link in the command line.
```
Comment (xofbd) [inline README.md]: good, just make sure the numbering is all `1.`

File: weather_app/greeting.py [hunk 4]
```diff
@@ -28,9 +30,9 @@ def get_temperature(lat, lon):
     url_base = 'https://api.met.no/weatherapi/locationforecast/2.0/compact'
 
     response = requests.get(url_base, params={'lat': lat, 'lon': lon})
-    data = response.json()
+    data = response.json()['properties']['timeseries']
 
-    return data['properties']['timeseries'][0]['data']['instant']['details']['air_temperature']
+    return data[0]['data']['instant']['details']['air_temperature']
```
Comment (xofbd) [inline weather_app/greeting.py:39]: this appears to be a simpler way to get the temperate data!

**Review by xofbd — CHANGES_REQUESTED** [review]
See my comments. Several things.

1. You'll have to rebase to fix merge conflicts. Let me know if you have questions with this.
1. Looks like we both display the temperature in C and F. What do you think about a feature that displays either C or F based on the client's country?
1. I've added a contributing guideline, one of those things is to add yourself to the contributors section in the README.

File: README.md (hunk 3 above)
Comment (Tim-Abwao) [reply README.md]: Okay, I'll restore all the 1's I had replaced.

File: README.md (hunk 1 above)
Comment (Tim-Abwao) [reply README.md:15]: Thanks for the tip; this is so much better than manually refreshing numbers. 

I'll restore the 1's I had replaced.

**Comment by Tim-Abwao** [comment]
A future PR (#10) will continue with these features, more cleanly.

The branch I created this PR with got several merge conflicts. Since I was still learning to rebase, I ended up with several unnecessary commits. A fresh start would be better.

**Pull request closed without merging**